packages/maskbook/src/utils/native-rpc/Web.ts

```diff
@@ -0,0 +1,28 @@
+import type { WebviewAPIs } from './types'
+import Services from '../../extension/service'
+import { definedSocialNetworkWorkers } from '../../social-network/worker'
+import * as settings from '../../settings/settings'
+
+export const WebviewAPI: WebviewAPIs = {
+    web_echo: async (arg) => arg,
+    getDashboardURL: async () => browser.runtime.getURL(''),
```

Comment:
How about 
```suggestion
    getDashboardURL: async () => browser.runtime.getURL('index.html'),
```
App needs to load the entry page of dashboard, and the path `index.html` is configured by JS.
Or we could use a predefined path constant instead here